### ISSUE TITLE
Feature Request-data exported/downloaded feature

### ISSUE THREAD

**Issue by fengyedou** [efigence/redmine_issue_statistics#5]
Can the graphs provide export or download as .png/.jpg with the numbers it calculates function?
or Can there is diagram linked with graph with statistics for copy the data?

I mean for the data, now only can seen from graph, but if we want to put the data in the report, needs to input manually, and graph only can been a snapshot to attach in the report.

**Comment by swiatkiewicz** [comment]
@fengyedou, There is an API which returns JSON with all of this values.

**Comment by fengyedou** [comment]
I am not sure how to operate. Can you guide how to use API to get download function avaiable?

**Comment by swiatkiewicz** [comment]
your_url/statistics.json?key=your_api_key.
if you want switch page, then add params at the end of line ex.: ;page=1

**Comment by fengyedou** [comment]
@swiatkiewicz 
My original thought is 2 usage:

1.To get the graph downloaded as a image file with the statistics data, then for some report, we donot need to draw the graph, but can add graph with the plug-in, can it be available by current plug-in?
2.To get a report diagram to show the statistics data.

I am not sure if we catch the same meaning:)

**Comment by swiatkiewicz** [comment]
No. There is no funcitonality to get this graph to file or get a report.

**Comment by fengyedou** [comment]
Ok, if you have time, hope to get more features, thanks:)
